lib/util.js

```diff
@@ -1058,4 +1058,58 @@ exports._exceptionWithHostPort = function(err,
 process.versions[exports.inspect.custom] =
   (depth) => exports.format(JSON.parse(JSON.stringify(process.versions)));
 
-exports.promisify = internalUtil.promisify;
+
+// Callbackify section
+const CBFY_ERR = 'The last argument to the callbackified function must be a ' +
+                 'callback style function';
+
+
+function callbackifyOnRejected(reason, cb) {
+  // !reason guard inspired by BlueBird https://goo.gl/t5IS6M
+  // Because `null` is a special error value in callbacks which means "no error
+  // occurred". We error-wrap so the callback consumer can distinguish between
+  // "the promise rejected with null" or "the promise fulfilled with undefined".
+  if (!reason) {
+    const newRej = new Error(reason + '');
```

Comment:
@mcollina This is in the `if (!reason)` branch, so I am not sure what exactly you mean by that; could you be more specific?